docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -42,7 +42,7 @@ For more information, see the Doctrine connection configuration example in the [
 
 ## Field Types
 
-The following tags used to register Field Type features in the [service container](https://doc.ibexa.co/en/latest/guide/service_container) have been renamed:
+The following tags used to register Field Type features in the [service container](https://doc.ibexa.co/en/latest/guide/api/public_php_api.md#service-container) have been renamed:
```

Comment:
Broken link

Reply:
This link and the others will be fixed when this PR is merged (because the `releases` section uses absolute links, they only work after merging).